src/runtime-boot/init/src/main.rs

```diff
@@ -44,17 +41,22 @@ fn main() -> Result<(), Box<dyn Error>> {
     // Mount the image
     const SYS_MOUNT_FS: i64 = 363;
 
+    // Example envs. must end with null
+    let env1 = CString::new("TEST=1234").unwrap();
```

Comment:
Why use example env set to that particular value?

Reply:
I kept it here to see it works.